awsshell/compat.py

```diff
@@ -10,11 +10,13 @@
 if PY3:
     from html.parser import HTMLParser
     text_type = str
+    long_type = int
     from io import StringIO
     import dbm
 else:
     from HTMLParser import HTMLParser
     text_type = unicode
+    long_type = long
```

Comment:
Yeah, I think it might be okay to map the coral types 'long' and 'int' to just use `int` in both python 2 and 3.
This is the behavior in python 2:
```
int('99999999999999999999') => 99999999999999999999L
```